src/hooks/useLegacyUIURL.ts

```diff
@@ -14,20 +14,17 @@ export const useLegacyUIURL = (): string | null => {
 
   useEffect(() => {
     const legacyUIMap = {
-      [routes.version]: `${uiURL}/version/${id}`,
-      [routes.configurePatch]: `${uiURL}/patch/${id}`,
+      [`${routes.version}/*`]: `${uiURL}/version/${id}`,
+      [`${routes.configurePatch}/*`]: `${uiURL}/patch/${id}`,
       [routes.userPatches]: `${uiURL}/patches/user/${id}`,
-      [routes.task]: `${uiURL}/task/${id}`,
+      [`${routes.task}/*`]: `${uiURL}/task/${id}`,
       [routes.spawnHost]: `${uiURL}/spawn#?resourcetype=hosts`,
       [routes.spawnVolume]: `${uiURL}/spawn#?resourcetype=volumes`,
-      [routes.commits]: `${uiURL}/waterfall/${id}`,
+      [`${routes.commits}/:id`]: `${uiURL}/waterfall/${id}`,
```

Comment:
why don't the rest of the routes use a param slug like this one?

Reply:
They all do except the spawn routes